crates/bevy_winit/src/accessibility.rs

```diff
@@ -117,26 +120,31 @@ fn update_accessibility_nodes(
     )>,
     node_entities: Query<Entity, With<AccessibilityNode>>,
 ) {
-    let Ok((primary_window_id, primary_window)) = primary_window.get_single() else {
+    let Ok((primary_window_entity, primary_window)) = primary_window.get_single() else {
+        return;
+    };
+    let Some(adapter) = adapters.get(&primary_window_entity) else {
         return;
     };
-    let Some(adapter) = adapters.get(&primary_window_id) else {
+    let Some(primary_window_id) = windows.entity_to_winit.get(&primary_window_entity) else {
```

Comment:
Small stylistic suggestion:
```suggestion
    let Some(&primary_window_id) = windows.entity_to_winit.get(&primary_window_entity) else {
```

This way you don't need to deref it like this `(*primary_window_id)`.